src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -160,89 +99,48 @@ class Comment extends React.Component<Props, State> {
                                 targetAttachment="bottom right"
                             >
                                 <Media.Menu isDisabled={isConfirmingDelete} data-testid="comment-actions-menu">
-                                    {canEdit && (
```

Comment:
I thought we discussed leaving this in for now due to possible upcoming feature work?